src/api/login.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export const validateLogin = (mail, password) => {
+  const HOST = process.env.REACT_APP_BASE_URL;
+  return axios
+    .post(`${HOST}/login`, {
+      mail,
+      password,
+    })
+    .then(function (response) {
+      return response.data;
+    })
+    .catch(function (error) {
+      console.log(error.message);
+      return error.response.data;
+    });
+};
```

Comment:
inside of api folder create a new file and name it `axios.config.js` and inside of that file 
```javascript
    const HOST = process.env.REACT_APP_BASE_URL; 
    const axiosInstance = axios.create({
        baseURL: HOST,
    });
    // TODO add interceptors when it is needed
    export { axiosInstance };
```

and then inside of the `login.js` file you will import the `axiosInstance`
```suggestion
import { axiosInstance } from './axios.config';

export const validateLogin = async (mail, password) => {
  
  return axiosInstance
    .post(`/login`, {
      mail,
      password,
    })
    .then(function (response) {
      return response.data;
    })
    .catch(function (error) {
      return error.response.data;
    });
};
```